server/routerlicious/packages/services-telemetry/src/resources.ts

```diff
@@ -38,3 +39,20 @@ export interface ILumberjackSchemaValidationResult {
 export interface ILumberjackSchemaValidator {
     validate(props: Map<string, any>): ILumberjackSchemaValidationResult;
 }
+
+export function handleError(eventName: LumberEventName, errMsg: string, engineList: ILumberjackEngine[]) {
+    if (process.env.NODE_ENV === "production")
+    {
+        if (engineList.length === 0) {
+            console.log(errMsg);
```

Comment:
Should `debug` be used here instead? Or if we're fully switching to Lumber from debug + winston, maybe having a ConsoleLumberjackEngine that you pass into a Lumber here instead of directly using `console.log`?

I mostly ask because there is only 1 other place in server/routerlicious where console.log is used ([getOrCreateRepository](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/services-client/src/utils.ts))

Reply:
@znewton and I discussed offline and the plan is to substitute `debug` eventually by Lumberjack - which offers a "debug" log level. Also, `console.log` here is used as a last resort. In production, we don't want to make the process crash by throwing errors due to the telemetry framework. So instead, we log those errors. In the extreme case that a LumberjackEngine has not been provided and the list is empty, we can't use a new Lumber as there is no engine. So, instead, we log a raw message to the console, so the message is still available in raw logs.